.run/Build system-probe.run.xml

```diff
@@ -13,7 +13,7 @@
     <option name="EXECUTE_SCRIPT_FILE" value="true" />
```

Comment:
@DataDog/ebpf-platform what about you co-owning this file with us, since you own the underlying task to build system-probe?